RELEASE.md

```diff
@@ -43,8 +43,7 @@ For a **PATCH** release:
 - Use `git` to cherry-pick all relevant PRs into your branch.
 - Update `pkg/generator/main.go` with the new semver tag and any updates to the API review URL.
 - Run the following command `BASE_REF=vmajor.minor.patch make generate` which will update generated docs
-  and webhook with the correct version info (Note that you can't test with these YAMLs yet until a tag
-  is created in later steps as they contain references to elements which wont exist until the tag is cut).
+  and webhook with the correct version info (Note that you can't test with these YAMLs yet as they contain references to elements which wont exist until the tag is cut and image is promoted to production registry).
```

Comment:
Nit: mind line wrapping at ~80 chars to match the rest? Also think that you may want a `.` after "info". Same comment applies to similar lines below.